### PR TITLE
Feature/telescope availability vue

### PR DESCRIPTION
**Pull request by jnation3406**

### CODE REVIEW

File: static/js/telescope_availability_app.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import Vue from 'vue';
+import $ from 'jquery';
+import {siteCodeToName} from './utils.js';
```
Comment (Fingel) [inline static/js/telescope_availability_app.js]: You can put these on the same line. {siteCodeToName, observatoryCodeToNumber} etc.

File: static/js/telescope_availability_app.vue [hunk 2]
```diff
@@ -0,0 +1,31 @@
+<template>
```
Comment (Fingel) [inline static/js/telescope_availability_app.vue]: I don't see the need to have this vue file and telescope_availability_app.js. Just merge this into the .js file and keep the component separate.

File: static/js/telescope_availability_app.vue [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import Vue from 'vue';
+import $ from 'jquery';
+import telescope_availability from './components/telescope_availability.vue';
+
+export default {
+  name: 'app',
+  components: {telescope_availability,},
+  data: function(){
+    return {
+      availabilityData2: {},
```
Comment (Fingel) [inline static/js/telescope_availability_app.vue]: Where is availabilityData1?

File: static/js/telescope_availability_app.vue (hunk 2 above)
Comment (jnation3406) [inline static/js/telescope_availability_app.vue]: I was following the example of request_details and compose, which each have a .vue and a .js.

Comment (Fingel) [inline static/js/telescope_availability_app.vue]: Both of those have javascript that has nothing to do with vue.js in them as well. But it's not a big deal. Can you remove the _app from all the files, we don't do that anywhere else.

Comment (jnation3406) [inline static/js/telescope_availability_app.vue]: okay, is chart better?

Comment (Fingel) [inline static/js/telescope_availability_app.vue]: yes

**Pull request merged**